pkg/apitype/updates.go

```diff
@@ -70,6 +70,12 @@ type UpdateMetadata struct {
 	Environment map[string]string `json:"environment"`
 }
 
+// LocalPolicyPack contains information about the locally specified Policy Packs ran with
+// an update.
+type LocalPolicyPack struct {
```

Comment:
So the `LocalPolicyPack` struct here isn't including the `Name` property like the `engine` package's `LocalPolicyPack` since this gets generated and set to the service before we start the plug-ins and can read the actual policy pack names, right? Just want to make sure this isn't an unintentional omission or something.

Reply:
This was an struct. Removed it.